frontend/public/locales/en/personal-information.json

```
@@ -67,7 +67,7 @@
       },
       "update-message": "Please update your phone number below.",
       "button": {
-        "save": "Save",
+        "save": "Change",
```

I think "submit" key would be better. You can make that change globally in a nother PR.
```suggestion
        "submit": "Change",
```